apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -4764,9 +4764,57 @@ sc_ws_contract_(Config, TestName, Owner) ->
     contract_calls_(TestName, ContractPubKey, Code, AckConnPid, UpdateVolleyReverse,
                     GetDecodedResult, SenderConnPid, AckPubkey, SenderPubkey),
 
+    GetPoI =
+        fun(ConnPid) ->
+            ?WS:send_tagged(ConnPid, <<"get">>, <<"poi">>,
+                            #{contracts   => [aec_base58c:encode(contract_pubkey, ContractPubKey)],
+                              accounts    => [aec_base58c:encode(account_pubkey, SenderPubkey),
+                                              aec_base58c:encode(account_pubkey, AckPubkey)]
+                            }),
 
-    ok = ?WS:unregister_test_for_channel_events(SenderConnPid, [sign, info, get]),
-    ok = ?WS:unregister_test_for_channel_events(AckConnPid, [sign, info, get]),
+                    {ok, <<"poi">>, #{<<"poi">> := P}} = ?WS:wait_for_channel_event(ConnPid, get),
+                    P
+                end,
+
+    GetMissingPoI =
+        fun(ConnPid, Accs, Cts) ->
+            ?WS:send_tagged(ConnPid, <<"get">>, <<"poi">>,
+                            #{contracts   => [aec_base58c:encode(contract_pubkey, C) || C <- Cts],
+                              accounts    => [aec_base58c:encode(account_pubkey, Acc) || Acc <- Accs]
+                            }),
+
+                    {ok, #{<<"reason">> := R}} = ?WS:wait_for_channel_event(ConnPid, error),
+                    R
+                end,
+
+    EncodedPoI = GetPoI(SenderConnPid),
+    EncodedPoI = GetPoI(AckConnPid),
+
+    NegativePoiTests =
```

Comment:
Perhaps these should be broken out into a separate test? Could also be done some other time.

Reply:
Tracked [SC WS Tests: revisit tests](https://www.pivotaltracker.com/story/show/159459719)